pub_sub/csharp/order-processor/Program.cs

```diff
@@ -0,0 +1,36 @@
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+    app.UseDeveloperExceptionPage();
+}
+
+app.MapGet("/dapr/subscribe", () => {
+    var sub = new DaprSubscription(pubsub_name: "order_pub_sub", topic: "orders", route: "orders");
+    Console.WriteLine("Dapr pub/sub is subscribed to: " + sub);
+    return Results.Json(new DaprSubscription[]{sub});
+});
+
+app.MapPost("/orders", (DaprData<Order> request_data) => {

```

Comment:
parameters here should be `camelCase` (same as local variables). A .NET user will think a python dev wrote this code :)

Reply:
roger_That ;-).  This DaprData<Order> stuff is removed btw since we'll `app.UseCloudEvents`